setup.py

```diff
@@ -19,24 +19,63 @@
 """
 
 import setuptools
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
 
 # Get the long description from the README file
 long_description = (here / 'README.md').read_text(encoding='utf-8')
 
+
 setuptools.setup(
-    name="arc3o-ClimateClara", 
-    version="0.1",
-    author="Clara Burgard",
-    author_email="clara.burgard@gmail.com",
-    description="The Arctic Ocean Observation Operator for 6.9 GHz",
+	
+	#The project's name
+    name='arc3o',
+    
+    #The project's version 
+    version='0.1',
+    
+    #The project's metadata
+    author='Clara Burgard',
+    author_email='clara.burgard@gmail.com',
+    description='An observation operator for the Arctic Ocean for 6.9 GHz',
     long_description=long_description,
-    long_description_content_type="text/markdown",
-    url="https://github.com/ClimateClara/arc3o",
-    packages=setuptools.find_packages(),
+    
+    #The project's main homepage.
+    url='https://github.com/ClimateClara/arc3o',
+    
+    #The project's license
     license='GPL-3.0',
+    
+    packages=find_packages(exclude=['docs', 'tests*', 'examples']),
+    
     classifiers=[
-        "Programming Language :: Python :: 3",
-        "Operating System :: OS Independent",
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Climate scientists',
```

Comment:
```suggestion
        'Intended Audience :: Science/Research',
```
Unfortunately the list of [available classifiers](https://pypi.org/classifiers/) is quite limited. An Climate Scientists is not in the list

Reply:
ok, did not know there was a list. I'll know better next time then!